benchmark/benchmark.py

```diff
@@ -167,12 +168,28 @@ def cli():
     # import the requested benchmark module
     mod = importlib.import_module(args.benchmark)
 
+    # check for additional device args
+    devs = copy(args.device)
```

Comment:
Needed to pass an additional kwarg to the device from the command line.

In future, I recommend we refactor the benchmarking suite to not be so rigid.

Reply:
agree. Should we make an 'catch-all' github issue to list improvements we've noticed? Device arguments, qnode attribute errors, etc.